public/index.html

```diff
@@ -0,0 +1,370 @@
+            <p>UI design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-8">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/koreanbeef.jpg" alt="Menu screenshot"></a>
+          <figcaption>
+            <h4>Korean beef</h4>
+            <p>Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+        <figure class="thumbnail  gallery-item-flex-9">
+          <a class="icon-see-more"  href="#imagine-informative-link-text-here-for-accessibility"><img src="images/thumbs.jpg" alt="Like dislike screenshot"></a>
+          <figcaption>
+            <h4>Facebook thumbs</h4>
+            <p>Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+      </section>
+      
+    </article>
+    <span class="white-notch"></span>
+    <!-- about  -->
+    <article class="about">
+      <h1>ABOUT</h1>
+      <section class="flex-container flex-paragraphs">
+        <p class="flex-paragraph-2">
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+        </p>
+        <p class="flex-paragraph-3">
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+        </p>
+      </section>
+      <h2>SKILLS</h2>
+      <section class="progressbars">
+        <figure>
+          <figcaption>PHOTOSHOP</figcaption>
+          <progress max="100" value="75"></progress>
+        </figure>
+        <figure>
+          <figcaption>HTML / CSS</figcaption>
+          <progress max="100" value="90"></progress>
+        </figure>
+        <figure>
+          <figcaption>ILLUSTRATOR</figcaption>
+          <progress max="100" value="65"></progress>
+        </figure>
+        <figure>
+          <figcaption>PHP</figcaption>
+          <progress max="100" value="55"></progress>
+        </figure>
+        <figure>
+          <figcaption>WORDPRESS</figcaption>
+          <progress max="100" value="95"></progress>
+        </figure>
+      </section>
+      <h3>Do you want to be one of the Avengers? Join us!</h3>
+      <button>CONTACT US</button>
+
+    </article>
+    <span class="black-notch"></span>
+    <!-- team  -->
+    <article class="team">
+      <h1>TEAM</h1>
+      <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+      <section class="team-member">
```

Comment:
Unnecessary `section`. 

Refer to [the spec](http://www.w3.org/html/wg/drafts/html/master/semantics.html#use-div-for-wrappers):

> A general rule is that the section element is appropriate only if the element’s contents would be listed explicitly in the document’s outline.